MoltenVK/MoltenVK/Utility/MVKBaseObject.mm

```diff
@@ -100,7 +100,7 @@
 	va_end(origArgs);
 
 	// Log the message to the standard error stream
-	if (shouldLog) { fprintf(stderr, "[%s] %s\n", getReportingLevelString(logLevel), pMessage); }
+	if (shouldLog) { fprintf(stdout, "[%s] %s\n", getReportingLevelString(logLevel), pMessage); }
```

Comment:
This seems wrong, and the old way (logging to `stderr`) right. Typically in Unix programming, actual program output goes to `stdout` while diagnostics go to `stderr`. Also, Wine logs to `stderr`; changing ours will make capturing MoltenVK logs alongside Wine logs harder.

Reply:
Yeah. Makes sense. Thanks for the convention background.

I had changed it because for some reason `stderr` wasn't being logged from background threads, but `stdout` was. But not sure why, or whether that was an appropriate solution, so I've reverted it to match convention.